### ISSUE TITLE
Fn wrapper in C# 9.0.

### ISSUE THREAD

**Issue by DiscreteTom** [DiscreteTom/UniStart#5]
From https://github.com/DiscreteTom/UniUtils/issues/1

Implement for `Entry`/`CBC`, so we can use it directly.

```cs
public class CBC {
  public static UnityAction Fn(UnityAction action) {
    return action;
  }
}

public class MyClass : CBC {
  void Start() {
    var f = Fn(()=>print(1));
  }
}
```

**Comment by DiscreteTom** [comment]
Implemented in v3.0.0.

https://github.com/DiscreteTom/UniStart/commit/8749caa8e31310e3508210c063ddb61bfcbd5ad4

**Comment by DiscreteTom** [comment]
Besides `Action`, `Func` with return type should also be implemented.
